testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -45,7 +45,21 @@ and adapter are all in the same JVM and you can debug them easily. If it is not
    
 and you will be able to attach remote debugger to the test. Unfortunately server and adapter are running in different JVMs, so this won't help to debug those. 
 
-TODO: Improve and add more info about Wildfly debugging...
+### JBoss auth server debugging
+
+When tests are run on JBoss based container (WildFly/EAP) there is maven profile which can be activated by 
+
+    -Pdebug-auth-server 
+
+By default the server will wait on startup to attach the debugger on localhost:5005. There are some properties what can change default behaviour.
```

Comment:
Can we have the reverse logic employed here? That by default the server would _not_ wait?